src/Analyzers/CSharp/Analyzers/ConvertNameOf/CSharpConvertNameOfDiagnosticAnalyzer.cs

```diff
@@ -34,15 +35,15 @@ public CSharpConvertNameOfDiagnosticAnalyzer()
 
         protected override void InitializeWorker(AnalysisContext context)
         {
-            context.RegisterSyntaxNodeAction(AnalyzeSyntaxAction, SyntaxKind.TypeOfExpression);
+            context.RegisterOperationAction(AnalyzeTypeOfAction, OperationKind.TypeOf);
         }
 
-        private void AnalyzeSyntaxAction(SyntaxNodeAnalysisContext syntaxContext)
+        private void AnalyzeTypeOfAction(OperationAnalysisContext typeofOp)
```

Comment:
Nit: We avoid abbreviated names for locals.

```suggestion
        private void AnalyzeTypeOfAction(OperationAnalysisContext context)
```